DataFormats/HGCalReco/interface/TICLLayerTile.h

```diff
@@ -14,6 +14,8 @@ class TICLLayerTileT {
     tile_[globalBin(eta, phi)].push_back(layerClusterId);
   }
 
+  int getTypeT() const { return T::type; }
```

Comment:
drop "get" prefix, please, to align with other method style

Reply:
done